includes/RplusGoogleAnalytics.php

```diff
@@ -35,12 +35,14 @@ public static function get_google_api_client() {
             return false;
         }
 
-        $path_autoloader = dirname( __FILE__ ) . '/vendor/autoload.php';
-        if ( ! file_exists( $path_autoloader ) ) {
-            return false;
-        }
+        if ( ! class_exists( 'Google_Client' ) ) {
+            $path_autoloader = dirname( __FILE__ ) . '/vendor/autoload.php';
+            if ( ! file_exists( $path_autoloader ) ) {
+                return false;
+            }
 
-        require_once $path_autoloader;
+            require_once $path_autoloader;
+        }
```

Comment:
Took a while to figure out why I got a fatal error here... Turns out it was because of #6. Still not sure about the `class_exists()` check, we should probably also check if ` Google_Client::LIBVER` is => `2.0.0`.